vcd/resource_vcd_network_routed.go

```diff
@@ -186,15 +186,15 @@ func resourceVcdNetworkRoutedCreate(d *schema.ResourceData, meta interface{}) er
 		Configuration: &types.NetworkConfiguration{
 			FenceMode: "natRouted",
 			IPScopes: &types.IPScopes{
-				IPScope: types.IPScope{
+				IPScope: []*types.IPScope{&types.IPScope{
```

Comment:
Why are we making this an array if we're then using **only** the first occurrence just a few lines below?

Reply:
In general we not making array, we are requested to provide array by vCD API.